sdk/tables/azure-data-tables/src/main/java/com/azure/data/tables/implementation/TablesConstants.java

```diff
@@ -68,6 +68,11 @@ public final class TablesConstants {
     public static final Set<String> METADATA_KEYS = Collections.unmodifiableSet(new HashSet<>(
         Arrays.asList(ODATA_EDIT_LINK_KEY, ODATA_ETAG_KEY, ODATA_ID_KEY, ODATA_METADATA_KEY, ODATA_TYPE_KEY)));
 
+    /**
+     * Scope for Cosmos endpoints.
+     */
+    public static final String COSMOS_SCOPE = "https://cosmos.azure.com/.default";
```

Comment:
Maybe we could also move `STORAGE_SCOPE` here

Reply:
@vcolin7 Storage has an entire file dedicated to its constants. I'm not sure why that was the original design choice, but since the ask for Cosmos Entra is time sensitive, I think it makes sense to leave any refactoring to a separate consolidation PR.

Reply:
That design choice was "just copy the Storage libraries". Not strong need to change it for now